robot_skills/src/robot_skills/arms.py

```diff
@@ -341,22 +373,28 @@ def handover_to_robot(self, timeout=10):
             return False
 
     def handshake(self, timeout=10):
-        return self._send_joint_trajectory([
-            [-0.2, 0.4, 0.7, 1.4, -1.75, 0.3, 0],
-            [-0.2, 0.4, 0.7, 1.6, -1.75, 0.3, 0],
-            [-0.2, 0.4, 0.7, 1.3, -1.75, 0.3, 0],
-            [-0.2, 0.4, 0.7, 1.4, -1.75, 0.3, 0]], timeout=rospy.Duration(timeout))
+        """
+        Perform a handshake
+        :param timeout: timeout in seconds
+        :return: True or False
+        """
+        return self.send_joint_trajectory('handshake', timeout)
 
     def _send_joint_trajectory(self, joints_references, timeout=rospy.Duration(5), joint_names = None):
-        '''
+        """
         Low level method that sends a array of joint references to the arm.
 
         If timeout is defined, it will wait for timeout*len(joints_reference) seconds for the
         completion of the actionlib goal. It will return True as soon as possible when the goal
         succeeded. On timeout, it will return False.
-        '''
+        :param joints_references: list of joint configurations,
+        which should be a list of the length equal to the number of joints to be moved
+        :param timeout: timeout for each joint configuration in seconds
```

Comment:
In general, it might be a good idea to add to the docstring what happens in case of a timeout of 0.0